src/tracktor/tracker.py

```diff
@@ -411,6 +428,11 @@ def step(self, blob, frame=1):
                                     additional_dets=other_pedestrians_bboxes,
                                     early_stopping=self.finetuning_config['early_stopping_classifier']
                                 )
+                            elif self.finetuning_config["build_up_training_set"]:
+                                track.update_training_set_classification(self.finetuning_config['batch_size'],
```

Comment:
it would be more memory efficient to store all bboxes accessible and sortable by track id in the tracktor object and just pass is to the track for finetuning